Lib9c/Action/RewardGold.cs

```diff
@@ -135,6 +137,7 @@ public IAccountStateDelta PrepareNextArena(IActionContext ctx, IAccountStateDelt
                     var addressList = states.TryGetState(listAddress, out List rawList)
                         ? rawList.ToList(StateExtensions.ToAddress)
                         : new List<Address>();
+                    var serializedList = rawList ?? List.Empty;
```

Comment:
`serializedList` is too general and ambigious naming. how about `nextAddresses` ?